resources/lang/en/users.php

```diff
@@ -53,6 +53,7 @@
         '404' => 'User not found! ;_;',
         'age' => ':age years old',
         'current_location' => 'Currently in :location',
+        'dateFormat' => 'MMMM YYYY',
```

Comment:
I don't think it's user specific so it should be in common. The syntax is momentjs specific so I think it should say it somewhere in the key.